### PR TITLE
config_log() includes babylon and NONMEM versions

### LINKED ISSUE #115: NONMEM and bbi versions in config_log
# Summary

As a user I would like to have some additional fields extracted from `bbi_config.json` into the `config_log` output:

* bbi_version
* nonmem_version

# Tests

### PR DESCRIPTION
**Pull request by copernican**
This PR closes #115. `bbi_version` and `nm_version` are added to the tibble returned by `config_log()`, and give the versions of babylon and NONMEM, respectively, used for the last model run.

The build is failing for the same reason as #148. Once the golden files have been updated, we should

1. Review and merge #148. This PR is branched off `feature/cfg-log-status`, which is the branch associated with #148.

2. Pull the updated golden files into this branch, which should result in a successful build.

3. Bump the version.

4. Review and merge this PR.

### CODE REVIEW

**Comment by copernican** [comment]
@seth127 this is ready for your review. Note that `bbi_version` and `nm_version` are handled differently because one is straightforward and the other is not.

File: tests/testthat/setup-workflow-ref.R [hunk 1]
```diff
@@ -183,3 +183,24 @@ perturb_file <- function(path, envir = parent.frame()) {
...
+#' elements set to `NA`.
+#'
+#' @param x The scalar value to repeat.
+#' @param i Integer vector of indices that should be `NA`.
+#' @param len Non-negative integer giving the desired length of the output
+#'   vector.
+#'
+#' @return A vector of length `len` whose elements
+rep_missing <- function(x, i, len) {
+  checkmate::assert_scalar(x)
```
Comment (seth127) [inline tests/testthat/setup-workflow-ref.R:199]: You should definitely learn Go. Type safety appears to be a passion of yours. Go shares this passion. Also, I need someone to code review my babylon PR's.

File: tests/testthat/test-config-log.R [hunk 2]
```diff
@@ -163,12 +177,37 @@ withr::with_options(list(rbabylon.model_directory = NULL), {
...
+      )
+    )
 
     # config log fields
-    expect_identical(log_df$data_md5, c(CONFIG_DATA_MD5, NA_character_, NA_character_, CONFIG_DATA_MD5))
-    expect_identical(log_df$data_path, c(CONFIG_DATA_PATH, NA_character_, NA_character_, CONFIG_DATA_PATH))
-    expect_identical(log_df$model_md5, c(CONFIG_MODEL_MD5, NA_character_, NA_character_, CONFIG_MODEL_MD5))
+    expect_identical(
+      log_df[["data_md5"]],
+      rep_missing(CONFIG_DATA_MD5, missing_idx, 4L)
```
Comment (seth127) [inline tests/testthat/test-config-log.R:193]: I like this helper

**Review by seth127 — APPROVED** [review]
LGTM

**Pull request merged**